examples/jsm/utils/TextureUtils.js

```diff
@@ -62,11 +62,33 @@ export function decompress( texture, maxTextureSize = Infinity, renderer = null
 
 	}
 
-	renderer.setSize( Math.min( texture.image.width, maxTextureSize ), Math.min( texture.image.height, maxTextureSize ) );
+	const width = Math.min( texture.image.width, maxTextureSize );
+	const height = Math.min( texture.image.height, maxTextureSize );
+
+	renderer.setSize( width, height );
 	renderer.clear();
+
+	const renderTarget = new WebGLRenderTarget( width, height, {
+		stencilBuffer: true,
+		colorSpace: IS_SRGB ? SRGBColorSpace : LinearSRGBColorSpace,
```

Comment:
```suggestion
		colorSpace: texture.colorSpace,
```
Wouldn't this be simpler?

Reply:
Agreed, if it works! If not we should probably include a comment explaining why `texture.colorSpace` is not used...

The texture for conversion may be non-color, using NoColorspace. Somewhat related:

- https://github.com/mrdoob/three.js/issues/26479#issuecomment-1759638903

@WestLangley should we consider NoColorSpace a valid color space for a render target? I think the answer is probably yes, but I'm not sure whether it's fully supported now.